media/css/mozorg/internet-health.scss

```diff
@@ -0,0 +1,836 @@
+        @include font-size(44px);
+        color: $colorInnovation;
+        content: '########################################';
+
+        @media #{$mq-tablet} {
+            @include font-size(52px);
+            line-height: 39px;
+        }
+
+        @media #{$mq-desktop} {
+            @include font-size(64px);
+            line-height: 48px;
+        }
+    }
+}
+
+#decentralization {
+    h2 span {
+        background: $colorDecentralization;
+    }
+
+    .content {
+        background-image: url('/media/img/mozorg/internet-health/decentralization.svg')
+    }
+
+    .footer-hr:after {
+        @include font-size(38px);
+        color: $colorDecentralization;
+        content: '[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[[';
+        letter-spacing: 0.18em;
+
+        @media #{$mq-tablet} {
+            @include font-size(50px);
+            line-height: 62px;
+        }
+
+        @media #{$mq-desktop} {
+            @include font-size(62px);
+            line-height: 78px;
+        }
+    }
+}
+
+#inclusion {
+    h2 span {
+        background: $colorInclusion;
+    }
+
+    .content {
+        background-image: url('/media/img/mozorg/internet-health/inclusion.svg')
+    }
+
+    .footer-hr:after {
+        @include font-size(46px);
+        color: $colorInclusion;
+        content: '<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<';
+        letter-spacing: 0.2em;
+
+        @media #{$mq-tablet} {
+            @include font-size(60px);
+            line-height: 37px;
+        }
+
+        @media #{$mq-desktop} {
+            @include font-size(78px);
+            line-height: 48px;
+        }
+    }
+}
+
+#literacy {
+    h2 span {
+        background: $colorLiteracy;
+    }
+
+    .content {
+        background-image: url('/media/img/mozorg/internet-health/literacy.svg')
+    }
+
+    .footer-hr:after {
+        @include font-size(28px);
+        color: $colorLiteracy;
+        content: '@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@';
+        letter-spacing: 0.26em;
+
+        @media #{$mq-tablet} {
+            @include font-size(36px);
+            line-height: 45px;
+        }
+
+        @media #{$mq-desktop} {
+            @include font-size(48px);
+            line-height: 60px;
+        }
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Newsletter
+
+#newsletter-subscribe {
+    background: #fff;
+    color: #000;
+    margin: 0;
+
+    a {
+        color: #000;
+    }
+
+    button,
+    input[type='email'] {
+        border: 1px solid #0c99d5;
+    }
+
+    button {
+        background: #00e6ff;
+        color: #4a4a4a;
+    }
+
+    .content {
+        padding-top: 0;
+    }
+
+    .form-title {
+        background-image: url('/media/img/mozorg/internet-health/newsletter.svg');
+    }
+
+    @media #{$mq-tablet} {
+        padding: 40px 0;
+
+        .content {
+            padding-top: 20px;
+        }
+    }
+}
+
+html[dir="rtl"] #newsletter-subscribe h3 {
```

Comment:
This right aligns both the initial h3 and the h3 on the thank you section (not quite sure if that's intentional here, as the other content on the initial form is still left aligned). Also, could this go in _newsletter.scss so the fix applies to other pages?